beta-src/src/components/ui/WDInfoPanelButton.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { useState } from "react";
+import WDButton from "./WDButton";
+
+interface WDInfoPanelButtonProps {
+  name: "Cancel" | "Draw" | "Pause";
+}
+
+const WDInfoPanelButton: React.FC<WDInfoPanelButtonProps> = function ({
+  name,
+}): React.ReactElement {
+  const [selected, setSelected] = useState(false);
+
+  const setSelection = (event) => {
+    event.preventDefault();
+    if (selected) {
+      setSelected(false);
+    } else {
+      setSelected(true);
+    }
+  };
```

Comment:
the state should be in the parent that uses this button. not sure if we even need this component...but if we do i would call it GameVoteButton or something like that, accept state and onClick as props, then bake in the checkmark for the vote

Reply:
Removed this component